cosmos/lambda_92000.json

```diff
@@ -0,0 +1,62 @@
+{
+  "rpc": "https://rpc.lambda.nodestake.org",
+  "rest": "https://rest.lambda.im",
+  "chainId": "lambda_92000-1",
+  "chainName": "Lambda",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lambda_92000/lambda.png",
+  "evm": {
+    "chainId": 92000,
+    "rpc": "https://evm.lambda.im"
+  },
+  "nodeProvider": {
+    "name": "NodeStake",
+    "email": "info@nodestake.top",
+    "website": "https://nodestake.top"
+  },
+  "stakeCurrency": {
+    "coinDenom": "LAMB",
+    "coinMinimalDenom": "ulamb",
+    "coinDecimals": 18,
+    "coinGeckoId": "lambda",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lambda_92000/lambda.png"
+  },
+  "walletUrl": "https://lambda.im/",
```

Comment:
@steveliupai Could you remove `walletUrl` field?

Reply:
done

Reply:
@HeesungB Anything else we need to fix?